src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -301,13 +307,25 @@ private InitCodeContext createInitCodeContext(
       MethodContext context,
       Collection<FieldConfig> fieldConfigs,
       InitCodeOutputType initCodeOutputType,
-      List<String> sampleCodeDefaultValues,
-      ImmutableMap<String, SampleParameterConfig> sampleParamConfigMap) {
+      SampleValueSet valueSet) {
     return InitCodeContext.newBuilder()
         .initObjectType(context.getMethodModel().getInputType())
         .suggestedName(Name.from("request"))
-        .initFieldConfigStrings(sampleCodeDefaultValues)
-        .sampleParamConfigMap(sampleParamConfigMap)
+        .initFieldConfigStrings(valueSet.getParameters().getDefaultsList())
+        .sampleParamConfigMap(
```

Comment:
It is fairly hard to understand what this large expression is doing. Maybe break it up instead of placing it all inline? Is the conversion from `SampleValueSet` to sampleParamConfigMap deserving of its own method?

Reply:
done.